manifest.json

```diff
@@ -5,5 +5,5 @@
   "description": "Auto-load a startup file with Obsidian Vim commands.",
   "author": "esm",
   "authorUrl": "",
-  "isDesktopOnly": true
+  "isDesktopOnly": false

```

Comment:
This plugin will fail to load if the Obsidian API version is too low; mobile support for vim-mode was only added _fairly_ recently if I'm not mistaken? I don't remember the API version that introduced it, but I think that this should probably set `minAppVersion` to ensure it doesn't crash on load.

Reply:
Vim mode got added mid-January. The latest public version should support it. I don't know the API version of the current public build though.

Reply:
Going through the patch notes it seems that vim-mode support was added in 0.13.24 on mobile.

Reply:
Got from [here](https://apps.apple.com/us/app/obsidian-connected-notes/id1557175442).

Reply:
Thanks, added.